kymatio/frontend/torch_frontend.py

```diff
@@ -0,0 +1,25 @@
+import torch.nn as nn
+
+class Scattering_torch(nn.Module):
+   def __init__(self, J, shape, max_order = None):
+       super(Scattering_torch, self).__init__()
+       self.J = J
+       self.shape = shape
+       self.max_order = max_order
+
+   def build(self):
+       self.filters = self.create_and_register_filters()
```

Comment:
So, the issue is that I'd like a similar code to remain: it's a way to remind the dev that one should call this function in build. I don't know the standard in python, but some reading I had on the internet about oop suggest to use a template-like mechanism.

Reply:
yeah I get why you did it. just pointing out that it will error.

In general, let's go for a minimal an amount of "oop" stuff as possible

What about the main question, calling `build` from `__init__`?

Reply:
Yes, but as the __init__ might changed, I don't know how to mention the ancestor of the object that its method "__init__" got overloaded. Or should it simply do nothing and still build itself?

Reply:
well so you are leaving `self.filters = ...` as an instruction to developers. In the same way I'd leave `self.build()` at the end of `__init__`

Reply:
I followed your guideline(please close if u're ok)

Reply:
Thanks